### ISSUE TITLE
Question: Custodian Lambda function logs and Outputs

### ISSUE THREAD

**Issue by sri321** [cloud-custodian/cloud-custodian#3317]
Hello,

i specified a s3 bucket while provisioning the custodian lambda policy, but the lambda function logs are missing in the bucket, 
can someone point out where i can find those logs ?

Thank you 
Sri

**Comment by kapilt** [comment]
at the moment logs for policy lambdas have to be explicitly set via `execution-options:` on the policy mode. I think we'll want to change this to automatically propagate cli options to the lambda policies.

**Comment by sri321** [comment]
do you mean like this ?

policies:
  - name: my-first-policy-cross-account
    mode:
        type: periodic
        schedule: "rate(1 day)"
        role: arn:aws:iam::123456789012:role/lambda-role
        execution-options:
          assume_role: arn:aws:iam::210987654321:role/target-role
          output_dir: s3-bucket-arn

**Comment by thisisshi** [comment]
agreed @kapilt the log/resource output behavior for lambda based policies is confusing on first usage.

another avenue to clarify this should be in the documentation for lambda policy usage as well as adding the output dir into the execution options by default considering the `--assume` arg is automatically passed into lambda functions currently
